docs/admin/hardware.rst

```diff
@@ -17,12 +17,25 @@ More information on hardware compatibility can be found on the `Qubes OS System
 
 In order to print submissions, a supported non-networked printer is required. We have tested and recommend the HP LaserJet Pro M404n. More printer options will be added in future releases.
 
-Lenovo T series Laptops
+.. _thinkpad_t_series:
+
+Lenovo T series laptops
 -----------------------
 
-Lenovo ThinkPad T490 (with 8th generation Intel Core processor)
+Lenovo ThinkPad T14 (with 10th-generation Intel Core processor)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ThinkPad T14 **with a 10th-generation Intel Core processor** is a recommended option for the SecureDrop Workstation. If you plan to use it:
+
+- If your laptop has come with Ubuntu preinstalled, run its **Software Updater** to update the BIOS automatically (via ``fwupd``).
+- Otherwise, follow the instructions below to ensure that the BIOS is up to date.
+
+Network devices (Ethernet and Wi-Fi) will not immediately work out of the box and require a one-time manual configuration on install. After Qubes starts for the first time, when ``sys-net`` fails to start, follow the instructions below for the :ref:`thinkpad_t490`.
```

Comment:
I only had to follow this procedure for the ethernet controller (`dom0:00_1f.6`), but I also installed Qubes 4.1, which has a newer kernel. Just to make sure, did you need to do it for both the wifi and ethernet devices on Qubes 4.0?

Reply:
This was my experience too; thanks for corroborating.  Clarified in ad6710183afe135216388aed55726f8fd895bc08.